src/Application/Injectors/Logging.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Injectors;
+
+use Monolog\Handler\RotatingFileHandler;
+use Monolog\Logger;
+use Monolog\Processor\MemoryPeakUsageProcessor;
+use Monolog\Processor\MemoryUsageProcessor;
+use Monolog\Processor\WebProcessor;
+use Ramsey\Uuid\Uuid;
+
+final class Logging
+{
+    /**
+     * Injects logging into the container
+     *
+     * @return mixed[]
+     *
+     */
+    public function __invoke(): array
+    {
+        $logger = new Logger('NARRATION');
```

Comment:
What do you think about adding the possibility to work with dynamic log channels in case you don't want to write everything in just one log file?

Reply:
Makes perfect sense to make this part more configurable. Should this be done via config files, different Injectors or something else? What is your design thoughts?